sample/shared/dynamic-features/feature1Impl/build.gradle.kts

```diff
@@ -65,11 +65,11 @@ compose.web.targets()
 
 plugins.removeAll { it is ComposeCompilerKotlinSupportPlugin }
 
-class ComposeNoNativePlugin : KotlinCompilerPluginSupportPlugin by ComposeCompilerKotlinSupportPlugin() {
+class ComposeNoNativePlugin : KotlinCompilerPluginSupportPlugin by ComposeCompilerKotlinSupportPlugin({}) {
```

Comment:
Can we use named arguments for the lamda, here and below?

Reply:
Sure thing. I hope the conventions / formatting match the one you use in the project. :)